contracts/azorius/BaseVotingBasisPercent.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity =0.8.19;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+/**
+ * An Azorius extension contract that enables percent based voting basis calculations.
+ *
+ * Intended to be implemented by BaseStrategy implementations, this allows for voting strategies
+ * to dictate any basis strategy for passing a Proposal between >50% (simple majority) to 100%.
```

Comment:
This could work anywhere from 0% to 100% right? Not just 50% to 100%?

Reply:
@tbwebb22 updated